BigBaseV2/src/views/view_weapons.cpp

```diff
@@ -27,6 +31,25 @@ namespace big
 
 			ImGui::Checkbox("No Spread", &g->weapons.no_spread);
 
+			if (ImGui::Button("Get All Weapons"))
+			{
+				QUEUE_JOB_BEGIN_CLAUSE()
+				{
+					for (auto const& weapon : weapon_list) {
+						WEAPON::GIVE_DELAYED_WEAPON_TO_PED(PLAYER::PLAYER_PED_ID(), weapon, 9999, false);
+					}
+					WEAPON::GIVE_DELAYED_WEAPON_TO_PED(PLAYER::PLAYER_PED_ID(), -72657034, 0, true);
+				}
+				QUEUE_JOB_END_CLAUSE
+			}
+
+			float damage_slider;
+			damage_slider = g->weapons.increased_damage;
+			if (ImGui::SliderFloat("Damage Multiplier", &damage_slider, 1.f, 10.f, "%.1f"))
+			{
+				g->weapons.increased_damage = damage_slider;
+			}
```

Comment:
Why aren't you directly assigning to g->weapons.increased_damage?
This is unnecessary.